Dockerfile

```diff
@@ -8,16 +8,18 @@ ENV MAGPIE_DIR=/opt/local/src/magpie
 ENV MAGPIE_ENV_DIR=$MAGPIE_DIR/env
 WORKDIR $MAGPIE_DIR
 
+ENV CRON_DIR=/etc/crontabs
+
 # magpie cron service
-COPY magpie-cron /etc/cron.d/magpie-cron
-RUN chmod 0644 /etc/cron.d/magpie-cron
-RUN touch ~/magpie_cron_status.log
-# set /etc/environment so that cron runs using the environment variables set by docker
-RUN env >> /etc/environment
+COPY magpie-cron $CRON_DIR/magpie-cron
+RUN chmod 0644 $CRON_DIR/magpie-cron
+RUN touch /var/log/cron.log
```

Comment:
Will not match existing variable `MAGPIE_CRON_LOG`.

Reply:
`/var/log/cron.log` is known at build time, while `MAGPIE_CRON_LOG` can be changed at runtime... I think these don't have to match? 
Maybe we just need one log file... I think MAGPIE_CRON_LOG wouldn't be necessary, what do you think?

Reply:
I don't mind removing it, but if it remains, it should be respected here and inside `magpie-cron` file.
Currently the variable is used here :
https://github.com/Ouranosinc/Magpie/blob/e8a47b0227910703201b4a159ad182c0270bd032/magpie/helpers/sync_resources.py#L336
So we would have different logging behaviours depending on which process runs the cron operation.

Reply:
I ended up keeping only `MAGPIE_CRON_LOG`